### ISSUE TITLE
Solving captchas in iframes error

### ISSUE THREAD

**Issue by yossinagar** [berstend/puppeteer-extra#120]
// Loop over all potential frames on that page
for (const frame of page.mainFrame().childFrames()) {
  // Attempt to solve any potential reCAPTCHAs in those frames
  await frame.solveRecaptchas()
}

get an error frame.solveRecaptchas() is not a function.

**Comment by berstend** [comment]
Can you paste in all relevant code please? Are you using `puppeteer-extra` and add the recaptcha plugin as documented?

**Comment by berstend** [comment]
Closing for now, please re-open if this is still an issue with the latest release.

**Comment by mikelam14** [comment]
I also encountered similar problem. I am solving recaptcha within the iframe, and I have located the correct frame. During debugging I think the Frame object has no solveRecaptchas() method at all, while the Page object still has. 

Code is something like

> `const puppeteer = require('puppeteer-extra');
const CaptchaPlugin = require('puppeteer-extra-plugin-recaptcha');
const captcha = CaptchaPlugin({
	provider: {
		id: '2captcha',
		token: 'xxxxx'
	},
	visualFeedback: true
});
puppeteer.use(captcha);

// found frame, name as myframe
await myframe.solveRecaptchas();`

It then says myframe has no method solveRecaptchas()

Thanks.
